docker/vpnspeed/nordvpn/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN echo "deb http://deb.debian.org/debian/ buster-backports main contrib non-fr
     DEBIAN_FRONTEND=noninteractive apt-get -q -y --no-install-recommends install openvpn wireguard
 RUN DEBIAN_FRONTEND=noninteractive apt-get -q -y autoremove --purge && \
     apt-get clean
-RUN mkdir /opt/wg && cd /opt/wg && git clone https://git.zx2c4.com/wireguard-tools && \
+RUN export GIT_SSL_NO_VERIFY=1 && mkdir /opt/wg && cd /opt/wg && git clone https://git.zx2c4.com/wireguard-tools && \
```

Comment:
What is the reason that we are not verifying TLS certificates here?

Reply:
This was introduced to mitigate expired certificate for explicitly this clone.